packages/blockchain-info-components/src/Form/SelectInput/template.js

```diff
@@ -26,15 +28,35 @@ const StyledSelect = styled(Select)`
   }
 `
 
-const customStyles = {
+const customStyles = theme => ({
   control: styles => ({
     ...styles,
-    backgroundColor: 'white',
+    backgroundColor: Color('white', theme),
```

Comment:
could this styles be moved under `StyledSelect` `.control`? connector won't be needed if it's possible

Reply:
The docs suggests this approach is best, also they recently merged this [PR](https://github.com/JedWatson/react-select/pull/2839) which will allow us to completely control the theme, but has not been released yet.

I don't think we'd need any css-in-js, which I think is less maintainable anyway.

Reply:
Done